---
description: Microsoft Edge WebView2 コントロールを使用して Win32 アプリの web コンテンツをホストする
title: 0.9.515-WebView2 (EdgeNotFoundException の場合)
author: MSEdgeTeam
ms.author: msedgedevrel
ms.date: 07/14/2020
ms.topic: reference
ms.prod: microsoft-edge
ms.technology: webview
keywords: IWebView2、IWebView2WebView、webview2、webview、win32 アプリ、win32、edge、ICoreWebView2、ICoreWebView2Controller、browser control、edge html
ms.openlocfilehash: db4a903ca430541fa9edec9d953bf28531f7c0a4
ms.sourcegitcommit: f6764f57aed9ab7229e4eb6cc8851d0cea667403
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 07/15/2020
ms.locfileid: "10879605"
---
# <span data-ttu-id="db96e-104">0.9.515 クラスの WebView2 クラス (EdgeNotFoundException)</span><span class="sxs-lookup"><span data-stu-id="db96e-104">0.9.515 - Microsoft.Web.WebView2.Core.EdgeNotFoundException class</span></span> 

> [!NOTE]
> <span data-ttu-id="db96e-105">この参照は、SDK バージョン0.9.515 後のリリースで変更される可能性があります。</span><span class="sxs-lookup"><span data-stu-id="db96e-105">This reference may be altered or unavailable for releases after SDK version 0.9.515.</span></span> <span data-ttu-id="db96e-106">最新 API リファレンスについては、 [WEBVIEW2 api リファレンス](../../../webview2-api-reference.md)を参照してください。</span><span class="sxs-lookup"><span data-stu-id="db96e-106">Please refer to [WebView2 API reference](../../../webview2-api-reference.md) for the latest API reference.</span></span>

<span data-ttu-id="db96e-107">名前空間: WebView2 () </span><span class="sxs-lookup"><span data-stu-id="db96e-107">Namespace: Microsoft.Web.WebView2.Core</span></span>\
<span data-ttu-id="db96e-108">アセンブリ: Microsoft.Web.WebView2.Core.dll</span><span class="sxs-lookup"><span data-stu-id="db96e-108">Assembly: Microsoft.Web.WebView2.Core.dll</span></span>

```
class Microsoft.Web.WebView2.Core.EdgeNotFoundException
  : public Exception
```

<span data-ttu-id="db96e-109">エッジのインストールが見つからない場合にスローされる例外。</span><span class="sxs-lookup"><span data-stu-id="db96e-109">The exception that is thrown when an Edge installation is missing.</span></span>

## <span data-ttu-id="db96e-110">まとめ</span><span class="sxs-lookup"><span data-stu-id="db96e-110">Summary</span></span>

 <span data-ttu-id="db96e-111">Members</span><span class="sxs-lookup"><span data-stu-id="db96e-111">Members</span></span>                        | <span data-ttu-id="db96e-112">説明</span><span class="sxs-lookup"><span data-stu-id="db96e-112">Descriptions</span></span>
--------------------------------|---------------------------------------------
[<span data-ttu-id="db96e-113">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-113">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="db96e-114">EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-114">Initializes a new instance of the EdgeNotFoundException class.</span></span>
[<span data-ttu-id="db96e-115">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-115">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="db96e-116">この例外の原因である内部例外への参照を使って、EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-116">Initializes a new instance of the EdgeNotFoundException class with a reference to the inner exception that is the cause of this exception.</span></span>
[<span data-ttu-id="db96e-117">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-117">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="db96e-118">指定したエラーメッセージを使用して、EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-118">Initializes a new instance of the EdgeNotFoundException class with a specified error message.</span></span>
[<span data-ttu-id="db96e-119">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-119">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="db96e-120">指定したエラーメッセージと、この例外の原因である内部例外への参照を指定して、EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-120">Initializes a new instance of the EdgeNotFoundException class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span>

## <span data-ttu-id="db96e-121">Members</span><span class="sxs-lookup"><span data-stu-id="db96e-121">Members</span></span>

#### <span data-ttu-id="db96e-122">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-122">EdgeNotFoundException</span></span> 

<span data-ttu-id="db96e-123">EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-123">Initializes a new instance of the EdgeNotFoundException class.</span></span>

> <span data-ttu-id="db96e-124">パブリック[EdgeNotFoundException](#edgenotfoundexception)()</span><span class="sxs-lookup"><span data-stu-id="db96e-124">public [EdgeNotFoundException](#edgenotfoundexception)()</span></span>

#### <span data-ttu-id="db96e-125">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-125">EdgeNotFoundException</span></span> 

<span data-ttu-id="db96e-126">この例外の原因である内部例外への参照を使って、EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-126">Initializes a new instance of the EdgeNotFoundException class with a reference to the inner exception that is the cause of this exception.</span></span>

> <span data-ttu-id="db96e-127">パブリック[EdgeNotFoundException](#edgenotfoundexception)(例外の内部)</span><span class="sxs-lookup"><span data-stu-id="db96e-127">public [EdgeNotFoundException](#edgenotfoundexception)(Exception inner)</span></span>

##### <span data-ttu-id="db96e-128">パラメーター</span><span class="sxs-lookup"><span data-stu-id="db96e-128">Parameters</span></span>
* `inner` <span data-ttu-id="db96e-129">現在の例外の原因である例外。</span><span class="sxs-lookup"><span data-stu-id="db96e-129">The exception that is the cause of the current exception.</span></span>

#### <span data-ttu-id="db96e-130">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-130">EdgeNotFoundException</span></span> 

<span data-ttu-id="db96e-131">指定したエラーメッセージを使用して、EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-131">Initializes a new instance of the EdgeNotFoundException class with a specified error message.</span></span>

> <span data-ttu-id="db96e-132">パブリック[EdgeNotFoundException](#edgenotfoundexception)(文字列メッセージ)</span><span class="sxs-lookup"><span data-stu-id="db96e-132">public [EdgeNotFoundException](#edgenotfoundexception)(string message)</span></span>

##### <span data-ttu-id="db96e-133">パラメーター</span><span class="sxs-lookup"><span data-stu-id="db96e-133">Parameters</span></span>
* `message` <span data-ttu-id="db96e-134">例外の理由を説明するエラーメッセージ。</span><span class="sxs-lookup"><span data-stu-id="db96e-134">The error message that explains the reason for the exception.</span></span>

#### <span data-ttu-id="db96e-135">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="db96e-135">EdgeNotFoundException</span></span> 

<span data-ttu-id="db96e-136">指定したエラーメッセージと、この例外の原因である内部例外への参照を指定して、EdgeNotFoundException クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="db96e-136">Initializes a new instance of the EdgeNotFoundException class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span>

> <span data-ttu-id="db96e-137">パブリック[EdgeNotFoundException](#edgenotfoundexception)(文字列メッセージ、例外の内部)</span><span class="sxs-lookup"><span data-stu-id="db96e-137">public [EdgeNotFoundException](#edgenotfoundexception)(string message, Exception inner)</span></span>

##### <span data-ttu-id="db96e-138">パラメーター</span><span class="sxs-lookup"><span data-stu-id="db96e-138">Parameters</span></span>
* `message` <span data-ttu-id="db96e-139">例外の理由を説明するエラーメッセージ。</span><span class="sxs-lookup"><span data-stu-id="db96e-139">The error message that explains the reason for the exception.</span></span> 

* `inner` <span data-ttu-id="db96e-140">現在の例外の原因である例外。</span><span class="sxs-lookup"><span data-stu-id="db96e-140">The exception that is the cause of the current exception.</span></span>

